docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,223 @@
+# **Feature Engineering for W&D (Wide and Deep Learning) using Friesian**
+With Friesian Table APIs, data scientists and machine learning engineers are able to quickly process datasets of all sizes to generate features for recommender models. This tutorial showcases how to extract features for [Wide and Deep Learning](https://arxiv.org/abs/1606.07792) using [movielens dataset](http://files.grouplens.org/datasets/movielens/).
+
+## **Key Concepts**
+**Wide and Deep Learning** is a noval recommender model proposed by Google. Specifically, A W&D model is jointly trained wide linear models and deep neural networks---to combine the benefits of memorization and generalization for recommender systems.
+Know more about W&D and it's implementation [here](https://github.com/intel-analytics/BigDL/tree/main/python/friesian/example/wnd)
+
+A **FeatureTable** is a distributed collection of data, it provides rich data processing and feature engineering methods for recommender systems.
```

Comment:
FeatureTable: (1) Distributed table for processing recsys feature 
                          (2) common tabular operations 
                          (3) high-level recsys feature transformation

Reply:
Add Orca Estimator